types/store_test.go

```diff
@@ -57,19 +56,19 @@ func (s *storeTestSuite) TestCommitID() {
 }
 
 func (s *storeTestSuite) TestNewTransientStoreKeys() {
```

Comment:
these may be duplicate tests, we may be able to drop them

Reply:
+1, could you check? Otherwise nice those tests in the store package.

Reply:
+1, we may need to move them to store pkg